### ISSUE TITLE
Inconsistent ordering of the output with "Script Dax"

### ISSUE THREAD

**Issue by olivergluevd** [TabularEditor/TabularEditor3#664]
Hi

### Problem 
When "Script Dax" is executed on a table, the ordering of the output is not _alphabetically sorted_.
It seems to be _sorted based on the creation time_. 

### Current Behavior 
(Example with Tabular Editor 3.3.5)
Empty model, start with Measure B:
![billede](https://user-images.githubusercontent.com/85997404/190984949-b670bc3f-0970-4393-88f2-d1ce90f71f3b.png)

Add a new Measure A:
![billede](https://user-images.githubusercontent.com/85997404/190984997-9facfca6-4e28-47b3-81e9-9898f3f3ea61.png)

Tabular Editor 3 will automatically sort to:
![billede](https://user-images.githubusercontent.com/85997404/190985061-521cc961-1c00-4a9e-8c01-62a3b470be94.png)

**Now you can achieve two different orderings for "Script Dax" by:**
(1) Press "Script Dax" on the Table:
![billede](https://user-images.githubusercontent.com/85997404/190985356-16cf8711-2c17-4665-9f1d-8a9c804cd9cd.png)
Output (_sorted based on the creation time_):
![billede](https://user-images.githubusercontent.com/85997404/190986548-181252ad-701d-4c92-ad58-dfba660cfb46.png)

(2) Select all measure in the Table and press "Script Dax":
![billede](https://user-images.githubusercontent.com/85997404/190986756-d1b009ed-5424-45e4-868a-b87baac3648b.png)
Output (_alphabetically sorted_):
![billede](https://user-images.githubusercontent.com/85997404/190986665-b6e3e5f8-7183-4471-9bfe-ae20a0c87725.png)

### Expected Behavior:
"Script Dax" is consistent in ordering the output as  _alphabetically sorted_.

**Comment by otykier** [comment]
Indeed, the DAX script order is currently determined by metadata order, which is typically based on object creation order. We will add an option to allow for alphabetical ordering when scripting.
